### DISCUSSION TITLE
Projects with low Build Activity - Survey

### DISCUSSION THREAD

**Discussion by jadsonjs** [vimeo/psalm#7214] [Psalm development (AKA internals)]
Dear Developers of project **vimeo/psalm**,

We are conducting a study on CI metrics and we noticed that in your project, something peculiar happened.

During the period you are using Travis CI, the project passed by a long period of inactivity.
We noticed that the TravisCI configuration file (.travis.yml) was committed to your project on the date of **2016-12-07**.
But there is only a build history in TravisCI for a very short period, between **2020-12-16** and **2021-01-11**, in which were performed suddenly **119** builds.

Please, we would like you to answer this short survey with just 4 questions to help us identify the cause of this behavior.

https://forms.gle/UkkFrxpi7MNscYmV7

**Comment by orklah** [comment]
Seems like a travis DB inconsistency.
When I check for random commits in 2019, I can find successful jobs flagged for Travis (for example, the first commit here: https://github.com/vimeo/psalm/commits/master?after=18c790eb99a5066dd60aeabd8e9bd7d6ccfb1f6b+5400&branch=master)

When clicking the link, travis says the job couldn't be found: https://travis-ci.org/github/vimeo/psalm/builds/563282936

I guess travis must have cleaned up their old runs at some point.
